test/testdraw.c

```diff
@@ -72,8 +72,8 @@ static void DrawPoints(SDL_Renderer *renderer)
         SDL_SetRenderDrawColor(renderer, 255, (Uint8)current_color,
                                (Uint8)current_color, (Uint8)current_alpha);
 
-        x = (float)(SDL_rand() % viewport.w);
-        y = (float)(SDL_rand() % viewport.h);
+        x = (float)(SDL_rand_n(viewport.w));
```

Comment:
Actually, we want integer coordinates, so you can just remove the extra parentheses.

Reply:
> Can you remove the extra parentheses?
> 
> This should probably be: x = SDL_rand_float() * viewport.w;

I think I've got all the parens fixed up. It's amazing how many extra ones you needed for the rand%n method.

I agree that some of the lines should probably be SDL_rand_float(). I'm unable to build the tests because of signing key issues, so I'm not comfortable making those changes to that code.